Fixes compiler warning:
 plat_linux_cdda.c:142: warning: implicit declaration of function 'free'
which prevents kdemultimedia to be built with FEATURES="strict stricter"

Added missing header file.

Signed-off-by: Peter Huewe <peterhuewe@gmx.de>
---

--- kdemultimedia-3.5.9/kscd/libwm/plat_linux_cdda.c
+++ kdemultimedia-3.5.9/kscd/libwm/plat_linux_cdda.c
@@ -44,6 +44,7 @@
 #undef inline
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <math.h>
 #include <sys/ioctl.h>
 #include <sys/types.h>
